mat64/matrix.go

```diff
@@ -359,7 +361,9 @@ func Det(a Matrix) float64 {
 	if a, ok := a.(Deter); ok {
 		return a.Det()
 	}
-	return LU(DenseCopyOf(a)).Det()
+	lu := &LU{}
```

Comment:
var lu LU (previous comment was supposed to be here - lu is not used as a non-receiver parameter).